com.unity.template-universal/Packages/com.unity.template.universal/package.json

```diff
@@ -7,6 +7,50 @@
   "host": "hub",
   "description": "The Universal Render Pipeline Template configures Project settings for Projects where performance, wide platform support, and ease of customizing graphics are the primary considerations.\n\nThis Template uses the Universal Render Pipeline (URP) and Shader Graph. URP is prebuilt Scriptable Render Pipeline that is quick and easy to customize, and lets you create optimized graphics across a wide range of platforms. URP also includes an optimized 2D renderer complete with 2D lights and pixel perfect rendering, and an integrated post-processing solution. Shader Graph is a tool that allows you to create shaders using a visual node editor instead of writing code.\n\nThis template contains a sample Scene that contains examples of how to configure lighting settings, Materials, Shaders, and post-processing effects in URP, several preconfigured Universal Render Pipeline Assets that let you quickly swap between graphics quality levels, and Presets that have been optimized for use with URP.",
   "dependencies": {
-    "com.unity.render-pipelines.universal": "13.0.0"
+	"com.unity.2d.sprite": "1.0.0",
+    "com.unity.2d.tilemap": "1.0.0",
+    "com.unity.ads": "3.7.1",
+    "com.unity.analytics": "3.6.11",
+    "com.unity.collab-proxy": "1.7.1",
+    "com.unity.ext.nunit": "1.0.6",
+    "com.unity.feature.development": "1.0.1",
+    "com.unity.purchasing": "3.2.2",
+    "com.unity.render-pipelines.universal": "13.0.0",
+    "com.unity.test-framework": "1.1.29",
+    "com.unity.textmeshpro": "3.0.6",
+    "com.unity.timeline": "1.6.2",
+    "com.unity.ugui": "1.0.0",
+    "com.unity.xr.legacyinputhelpers": "2.1.8",
```

Comment:
It feels to me this one doesn't belong here either. Is this intended?

Reply:
Not sure if com.unity.xr.legacyinputhelpers is required now. @ryanhy-unity  ?

Reply:
I think Thomas answered this in that this package can allow XR HMD head tracking when not using the new input system.  Its name is weird in that it uses legacy which is a bit of a misnomer I think.